tests/integration/requirements.yml

```diff
@@ -3,3 +3,7 @@ collections:
   - ansible.windows
   - ansible.utils # ipv6 filter
   - amazon.cloud # used by integration tests - rds_cluster_modify
+  - ansible.netcommon
```

Comment:
to be consistent with the Zuul configuration, the dependency should be defined using:

```
- name: https://github.com/ansible-collections/{{ namespace }}.{{ name }}.git
  type: git
  version: main
```